docs/saveBodyTemperature.md

```diff
@@ -0,0 +1,23 @@
+save a body temperature value to Healthkit
+
+`saveBodyTemperature` accepts an options object containing a value:
+
+```javascript
+let options = {
+  value: 96,
+  date: new Date(2016, 5, 1).toISOString(), // optional; default now
+  unit: 'celsius', // optional, default fahrenheit
+}
+```
+
+```javascript
+AppleHealthKit.saveBodyTemperature(
+  (options: Object),
```

Comment:
The types here should match the original types from the library. In this case, would be:

```typescript
  (options: HealthValueOptions),
  callback: (error: string, results: HealthValue) => {
    ...
```